doc/INSTALL

```diff
@@ -1,8 +1,8 @@
 Requirements
 -------------
 
-* Python 2.7 or later.
-* Python Twisted, Tornado or asyncio (For async client and server)
+* Python 3.7 or later.
+* Python asyncio (preferred), Twisted or Tornado (For async client and server)
```

Comment:
Python asyncio is builtin, so it may not be a requirement anymore.
For Twisted and Tornado  we should mention that stuff is in maintainance mode at the moment and other solutions (like plugins) may be provided in future or even dropped.

Reply:
ups, you are quite right, only excuse is that I have not had my coffee yet :-)

Added a commit.